### PR TITLE
Release notes for iOS ACPCore 2.7.2, ACPIdentity 2.3.2

### PR DESCRIPTION
**Pull request by emdobrin**

### CODE REVIEW

File: release-notes.md [hunk 1]
```diff
@@ -10,6 +10,23 @@ The Adobe Experience Platform Mobile SDK will soon be available in Swift as an i
 
 [Sign up](https://forms.microsoft.com/Pages/ResponsePage.aspx?id=Wht7-jR7h0OUrtLBeN7O4UJN9zAhIEhJr3PBfyMf9wdUQTI2S0pMVEVYS1k3UUNJVDNDWlRUTFk4Qi4u) for a free developer preview.
 
+## Aug 10, 2020
+
+### iOS Core 2.7.2 \(Released with ACPCore version 2.7.3 on Cocoapods\)
+
+* Fixed a crash happening on `AdobeMarketingMobile::RulesEngine::ProcessEventForRules()`.
+* Fixed an issue where null values in rules consequences was not respected.
```
Comment (sbenedicadb) [inline release-notes.md:18]: `was not respected` -> `were not respected`

File: release-notes.md [hunk 2]
```diff
@@ -10,6 +10,23 @@ The Adobe Experience Platform Mobile SDK will soon be available in Swift as an i
...
+
+### iOS Core 2.7.2 \(Released with ACPCore version 2.7.3 on Cocoapods\)
+
+* Fixed a crash happening on `AdobeMarketingMobile::RulesEngine::ProcessEventForRules()`.
+* Fixed an issue where null values in rules consequences were not respected.
+
+### iOS Identity 2.3.2
+
+* Identity shared state now gets updated in current session on server response changes for blob or locationHint.
+* In order to improve the Analytics push tracking reports, the push notification preferences (a.push.optin) are now forwarded to Analytics when `setPushIdentifier` ACPCore API is first called with nil, in addition to the push preferences change optin to optout or output to optin.
```
Comment (sbenedicadb) [inline release-notes.md:23]: feels a little wordy.  an alternative: 

    Push notification preferences (`a.push.optin`) are now forwarded to Analytics whenever the value passed to `setPushIdentifier` is different than the previous time it was called.

File: release-notes.md [hunk 3]
```diff
@@ -10,6 +10,23 @@ The Adobe Experience Platform Mobile SDK will soon be available in Swift as an i
...
+* Improved safety checks for the Identity APIs with completion handler.
```
Comment (sbenedicadb) [inline release-notes.md:24]: another alternative:

    Completion handler added to several Identity APIs allowing more control for the caller.

Comment (emdobrin) [reply release-notes.md:24]: We didn't add the completion handler in this release, we just added more health checks on them.

**Comment by emdobrin** [comment]
@praschetan @imironchuk ready for your review

**Pull request merged**